src/ansys/pyensight/core/dockerlauncher.py

```diff
@@ -452,8 +466,18 @@ def connect(self):
 
         use_egl = self._use_egl()
 
+        # get the environment to pass to the container
+        container_env_str = ""
+        if self._pim_instance is not None:
+            container_env = self._get_container_env()
+            for i in container_env.items():
+                container_env_str += f"{i[0]}={i[1]}\n"
+
         # Run EnSight
         ensight_env_vars = None
+        if container_env_str != "":
+            ensight_env_vars = container_env_str
+
         if use_egl:
             ensight_env_vars = "LD_PRELOAD=/usr/local/lib64/libGL.so.1:/usr/local/lib64/libEGL.so.1"
```

Comment:
This should be appended to current value, when not None.
```suggestion
            if ensight_env_vars is None:
                ensight_env_vars = "LD_PRELOAD=/usr/local/lib64/libGL.so.1:/usr/local/lib64/libEGL.so.1"
            else:
                ensight_env_vars += "LD_PRELOAD=/usr/local/lib64/libGL.so.1:/usr/local/lib64/libEGL.so.1"
```